sites/friday/src/increase-observability/application-metrics/blackfire.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Blackfire for PHP and Python
+weight: 5
+description: A full access to Blackfire is bundled with your PHP and Python {{< vendor/name >}} projects.
+---
+
+A full access to [Blackfire](https://www.blackfire.io/) is bundled with all your PHP and
+Python {{< vendor/name >}} projects.
+
+Blackfire is the **official {{< vendor/name >}} observability service** that helps you
+improve the performance of your apps at each stage of their lifecycle.
+With Blackfire's unique Application Performance Monitoring (APM), Profiling,
+Alerting, and Testing features, you can achieve the following goals:
+
+- Avoid performance bottlenecks by proactively identifying issues in your code
+- Promptly solve identified issues by taking advantage of actionable recommendations
+- Create performance budgets for critical parts of your app and get alerted of any
+  problem before a change hits your production
+
+Blackfire is installed natively on {{< vendor/name >}} and [works integrally with the {{< vendor/name >}} workflow](https://www.youtube.com/watch?v=Bq-LFjgD6L0).
+This results in an effortless setup process and smooth user experience.
+
+## Get started with Blackfire
+
+You can only access your Blackfire environments after you've been granted access to the related {{< vendor/name >}} project.
+Therefore, to access your Blackfire environments, make sure you log in using your {{< vendor/name >}} account.
+
+To access a Blackfire environment, each project user needs a Blackfire account.
+When a project user doesn't already have a Blackfire account,
+a new one is automatically created using the user's {{< vendor/name >}} credentials.
+
+{{< note >}}
+If you're using a [Content Delivery Network (CDN)](../../domains/cdn/_index.md),
+make sure you [configure it](https://blackfire.io/docs/integrations/proxies/index)
+to let Blackfire profile the code running on your servers.
+{{< /note >}}
+
+### Automated integration
+
+The Blackfire automated integration is enabled on your environments by default.
+
+When you create a new environment,
```

Comment:
I remember us not having a 1:1 on environments previously.

Every Upsun environment = a BF environment? Not an Upsun project = a BF environment?